tests/tck/features/bugfix/YieldVar.feature

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2021 vesoft inc. All rights reserved.
+#
+# This source code is licensed under Apache 2.0 License,
+# attached with Common Clause Condition 1.0, found in the LICENSES directory.
+Feature: Test yield var
+
+  Background:
+    Given a graph with space named "nba"
+
+  Scenario: yield constant after pipe
+    When executing query:
+      """
+      $var = GO FROM "Tim Duncan" OVER *; YIELD $var[0][0] as var00;
+      """
+    Then the result should be, in any order:
+      | var00           |
+      | "Manu Ginobili" |
+      | "Manu Ginobili" |
+      | "Manu Ginobili" |
+      | "Manu Ginobili" |
+      | "Manu Ginobili" |
+      | "Manu Ginobili" |
+      | "Manu Ginobili" |
```

Comment:
The result of $var[row][col] is not a cell?

Reply:
Yes, it's not constant expr.